conditional/__init__.py

```diff
@@ -62,23 +68,25 @@ def database_processor(logger, log_method, event_dict): # pylint: disable=unused
                 blueprint=event_dict['blueprint'],
                 path=event_dict['path'],
                 description=event_dict['event']
-                )
+            )
             db.session.add(log)
             db.session.flush()
             db.session.commit()
         del event_dict['request']
     return event_dict
 
+
 structlog.configure(processors=[
     request_processor,
     database_processor,
     structlog.processors.KeyValueRenderer()
-    ])
+])
 
 logger = structlog.get_logger()
 
+from conditional.util.auth import get_username
```

Comment:
I'd prefer if this was placed on line 116 with the other `conditional.util` import, but it's okay to leave if you feel that it should be placed here instead.